PowerEditor/src/NppIO.cpp

```diff
@@ -649,6 +649,14 @@ bool Notepad_plus::doSave(BufferID id, const TCHAR * filename, bool isCopy)
 			}
 
 		}
+		else
+		{
+			_nativeLangSpeaker.messageBox("FileLockedWarning",

```

Comment:
At first I thought this isn't localized. But turns out, there is a localization for `FileLockedWarning`.
But for some reason, it's not used anywhere in the code.
Anyway, I would recommend adding `MB_ICONWARNING`.